File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h = get_int("Height: \n"); //ultimate size of the pyramid
    int t = h; //used to track which iteration we are in. 

    while (t > 0)
    {
      
        for (int w = 0; w < (t-1); w++)
            {
                printf("#");
            }
        printf("\n");
        t = t-1;
    }
    printf("\n");
}

File: mario.py
from cs50 import get_int
from cs50 import get_string

while True:
    h = get_int("Height: ")
    if h > 0 and h < 9:
        break

t = h

while (t > 0):
    for b in range(t-1):
        print(" ", end="")
        
    w = (t - 1)
    
    while (w < h):
        print("#", end="")
        w += 1
    
    print()
    t -= 1
        
        